README.md

```diff
@@ -50,9 +50,11 @@ Memoization is easy to implement for code written in functional style; in many l
 
 See section [recursion](/recursion) for description.
 
-## Lamba Expressions
+## [Lambda Expressions](/lambda)
 
-*(Coming soon)*
+Lambda expressions, or anonymous functions, are not a concept unique to functional programming. However, functional programming uses lambda expressions in concepts.
+Lambda expressions are functions not bound to an identifier, a common example of this is in-line functions. In functional programming these are used
+in techniques such as currying, and high-order functions.
```

Comment:
Not sure if you want to put this in here, but the original Lambda expression was not a function at all--it was a textual expression of the form (x -> expression) where all instances of 'x' inside of 'expression' were replaced with the token that followed the expression.  So `(apply (lambda x : x + x) 3)` collapsed to an expression equivalent to `(3 + 3)`